extensions/schema-compare/src/test/utils.test.ts

```diff
@@ -115,7 +116,16 @@ describe('utils: In-depth tests to verify verifyConnectionAndGetOwnerUri', funct
 	});
 
 	it('Should throw an error for login failure', async function (): Promise<void> {
-		const getConnectionsResults: azdata.connection.ConnectionProfile[] = [{ ...mockConnectionProfile }];
+		const connectionProfile = azdataTest.stubs.connectionProfile.createConnectionProfile({
+			// these need to match what's in MockConnectionInfo in testUtils.ts
```

Comment:
If they need to match, can we reuse MockConnectionInfo then instead of having the same literals copied?

Reply:
good idea!